ci/tasks/parse-release-tag.yml

```diff
@@ -0,0 +1,16 @@
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: alpine
+inputs:
+  - name: git-release
+outputs:
+  - name: tags
+run:
+  path: sh
+  args:
+    - -ec
+    - |
+      RELEASE_NUMBER=$(cat git-release/.git/ref | sed 's/alpha_release-//')
+      echo "${RELEASE_NUMBER}-release" > tags
```

Comment:
I think you need to `cd tags` into the directory otherwise it will create it in the current working directory. Since the tags folder is pre-created it won't let you do echo "..." > tags as the tags name is already taken (I tried to do that on my Mac with creating a tags folder and doing echo "..." which it wouldn't let me do.)